Student/Student.xcodeproj/xcshareddata/xcschemes/Student.xcscheme

```diff
@@ -124,9 +124,9 @@
       </EnvironmentVariables>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
was this intentional to commit? or just part of your testing process?

Reply:
Accident, thanks